ntex/src/http/client/connect.rs

```diff
@@ -20,7 +20,7 @@ where
     }
 }
 
-pub(super) trait Connect: fmt::Debug {
+pub trait Connect: fmt::Debug {
```

Comment:
let's create "unstable" feature or at least hide it. 
I want to add middlewares to http client, I may need to refactor all this.